### PR TITLE
Fix FailedToParse errors during get_stats() for Consumer and Producer in rust API

### PR DESCRIPTION
**Pull request by Hartigan**
Update ```ProducerStat``` and ```ConsumerStat``` according to ```FillJsonStats``` methods in mediasoup C++ library.

1. https://github.com/versatica/mediasoup/blob/v3/worker/include/RTC/RtpStream.hpp#L202  - rtt is ```float```
2. https://github.com/versatica/mediasoup/blob/v3/worker/src/RTC/RtpStreamRecv.cpp#L230  - bitrateByLayer is optinal field

### CODE REVIEW

File: rust/src/router/consumer.rs [hunk 1]
```diff
@@ -239,7 +239,7 @@ pub struct ConsumerStat {
     pub packet_count: usize,
     pub byte_count: usize,
     pub bitrate: u32,
-    pub round_trip_time: Option<u32>,
+    pub round_trip_time: Option<f64>,
```
Comment (nazar-pc) [inline rust/src/router/consumer.rs]: This should be `f32` here and below, it is just `float` on C++ side

**Review by nazar-pc — CHANGES_REQUESTED** [review]
Nice catch!

**Review by nazar-pc — APPROVED** [review]
Nice, thank you!

File: rust/src/router/consumer.rs (hunk 1 above)
Comment (Hartigan) [reply rust/src/router/consumer.rs]: Changed ```f32```. Thanks

**Comment by Hartigan** [comment]
@nazar-pc  Can you please upload a new version with fixes to crates.io?

**Comment by nazar-pc** [comment]
Will do probably tomorrow, you can use `git = "https://github.com/versatica/mediasoup.git"` in the meantime

**Comment by nazar-pc** [comment]
Rust 0.8.5 is out

**Comment by Hartigan** [comment]
Thanks

**Pull request merged**